.github/ISSUE_TEMPLATE/support_new_asset.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Support new asset
+about: Request support for a new asset
+---
+
+<!-- You can request for official support of an asset by filling in this template. But you can also easily add support for it on your local copy by following this guide: https://rotki.readthedocs.io/en/latest/usage_guide.html#adding-editing-an-asset -->
+
+
+**Asset name:** Please fill in
+**Asset symbol:** Please fill in
+**Coingecko link:** Please fill in
```

Comment:
```suggestion
**Coingecko link:** Please fill in - if available
```

Reply:
If that's not available then the asset will be rejected by us.

Reply:
But it's enough if it has one of both (coingecko/cc), doesn't have to be coingecko

Reply:
fair enough. True.